imager/imager_images/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.conf import settings
+
+# Create your models here.
+
+
+@python_2_unicode_compatible
+class Album(models.Model):
+    name = models.CharField(max_length=30)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                            on_delete=models.CASCADE,
+                            related_name='albums',
+                            null=True)
+
+    def __str__(self):
+        return self.name
+
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+    img_name = models.CharField(max_length=30)
+    image = models.ImageField(upload_to='photo_files/%Y-%m-%d')
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, 
+                            on_delete=models.CASCADE,
+                            related_name='photos',
+                            null=True)
+    albums = models.ManyToManyField(Album, related_name='photos')
+
```

Comment:
"Meta-data should include a title and a description, a date_uploaded, date_modified and date_published field. You should also have a ‘published’ field which takes one of several possible values (‘private’, ‘shared’, ‘public’)"